spec/models/vote_spec.rb

```diff
@@ -1,8 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Vote, type: :model do
+  let!(:new_article) { create :article, title: 'Comment tester' }
+  let!(:upvote) { create :vote, score: 1, voteable: new_article }
+  let!(:downvote) { create :vote, score: -1, voteable: new_article }
```

Comment:
We haven't talked much about some of Factory Girl's other features, but it has some nice ways to make factories easier to use:

```ruby
FactoryGirl.define do
  factory :vote do
    user

    factory :upvote do
      score 1
    end

    factory :downvote do
      score -1
    end
  end
end
```

You can use these in your tests like this:

```ruby
let!(:upvote) { create :upvote, voteable: new_article }
let!(:downvote) { create :downvote, voteable: new_article }
```

Not required, but a good practice, since it keeps your implementation for votes (i.e. score integer where 1 is upvote and -1 is downvote) in a single place.

Factory Girl has a ton of cool stuff, so might be good to spend a bit of time reading through [the docs](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md).